File: C/1018.Banknotes.c
#include <stdio.h>

int main(void){
    int dinheiro;

    scanf("%d", &dinheiro);

    printf("%d\n", dinheiro);

    printf("%d nota(s) de R$ 100,00\n", dinheiro/100);
    dinheiro %= 100;
    printf("%d nota(s) de R$ 50,00\n", dinheiro/50);
    dinheiro %= 50;
    printf("%d nota(s) de R$ 20,00\n", dinheiro/20);
    dinheiro %= 20;
    printf("%d nota(s) de R$ 10,00\n", dinheiro/10);
    dinheiro %= 10;
    printf("%d nota(s) de R$ 5,00\n", dinheiro/5);
    dinheiro %= 5;
    printf("%d nota(s) de R$ 2,00\n", dinheiro/2);
    dinheiro %= 2;
    printf("%d nota(s) de R$ 1,00\n", dinheiro);

    return 0;
}

File: Python/1018.Banknotes.py
dinheiro = int(input())

print(dinheiro)

print(dinheiro//100, "nota(s) de R$ 100,00")
dinheiro %= 100
print(dinheiro//50, "nota(s) de R$ 50,00")
dinheiro %= 50
print(dinheiro//20, "nota(s) de R$ 20,00")
dinheiro %= 20
print(dinheiro//10, "nota(s) de R$ 10,00")
dinheiro %= 10
print(dinheiro//5, "nota(s) de R$ 5,00")
dinheiro %= 5
print(dinheiro//2, "nota(s) de R$ 2,00")
dinheiro %= 2
print(dinheiro//1, "nota(s) de R$ 1,00")